src/cmd/flux-job.c

```diff
@@ -3536,6 +3546,34 @@ int cmd_taskmap (optparse_t *p, int argc, char **argv)
     return 0;
 }
 
+int cmd_timeleft (optparse_t *p, int argc, char **argv)
+{
+    int optindex = optparse_option_index (p);
+    flux_t *h;
+    flux_error_t error;
+    double t;
+    char buf[64];
+
+    if (optindex < argc - 1) {
+        optparse_print_usage (p);
+        exit (1);
+    }
+    if (!(h = flux_open (NULL, 0)))
+        log_err_exit ("flux_open");
+
+    if (optindex < argc) {
+        if (setenv ("FLUX_JOB_ID", argv[optindex++], 1) < 0)
+            log_err_exit ("setenv");
+    }
+    if (flux_job_timeleft (h, &error, &t) < 0)
+        log_msg_exit ("%s", error.text);
+    if (fsd_format_duration (buf, sizeof (buf), t) < 0)
+        log_err_exit ("fsd_format_duration");
+    printf ("%s\n", buf);
```

Comment:
maybe can be for a follow up PR, but perhaps option to output in plain ole seconds w/o the fsd suffix?  I can imagine users wanting to `sec=$(flux job timeleft)` and then immediately use the seconds in something else time related ... or possibly just check for `-eq 0`.  i.e. not waste time converting from fsd to a units they care about.

Reply:
Yeah, @garlick and I talked about that, but there didn't seem to be a known use case at this point and adding more development and test effort at this point seemed like a poor choice.

Reply:
I don't know if there's an existing fsd <-> seconds convenience function.  Absent that, I personally would prefer seconds rather than fsd, because I can so easily convert seconds to minutes/hours/days but the reverse is not so easy.

That said, for SCR I'll be using python and using flux.job.timeleft(), so I have no concrete use case either.

Reply:
Ok, since there is two votes for seconds, I"ll go ahead and implement that, with a `-H, --human` option to return FSD.

What do people think should be returned for the `infinity` case: `-1` or `UINT_MAX`?

Reply:
I'd vote for UINT_MAX.  Then a batch script could do something like

```
thing_runtime=3600
while test $(flux job timeleft) -gt $thing_runtime; do
    flux mini run -t $thing_runtime $thing
done
```